### PR TITLE
donate

### PR DESCRIPTION
**Pull request by feroult**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,6 +18,12 @@ Currently it provides you with: a few utilities, images/sprites/sprite sheets, a
 
 You can use whatever ones you want, as they are all somewhat independent.
 
+## Donate
+
+Donate to support Flame's development:
+
+[![Donate](https://img.shields.io/badge/Donate-PayPal-green.svg)](https://www.paypal.com/cgi-bin/webscr?cmd=_donations&business=VJS3922363XNU&currency_code=USD&source=url)
+
```
Comment (luanpotter) [inline README.md]: Good idea! Let's create a Fireslime paypal account, though heheheh

Comment (feroult) [reply README.md]: Good!! :)

Comment (erickzanardo) [reply README.md]: Maybe this is a better idea: https://www.paypal.com/us/webapps/mpp/money-collection#

What do you think @luanpotter @feroult ?

Comment (luanpotter) [reply README.md]: Looks cool, but does it need a fixed target amount? Or can it stay open forever?

Comment (feroult) [reply README.md]: @luanpotter I think it should be open. (any cons?)

@erickzanardo never used that... but if it works....  :)

**Comment by erickzanardo** [comment]
Take a look on what I sent on our channel on Discord

Em qui, 7 de fev de 2019 08:16, Fernando Ultremare <notifications@github.com>
escreveu:

**Pull request merged**